scholarx/archive/functions.js

```diff
@@ -117,25 +97,25 @@ async function loadData() {
 }
 loadData();
 //this function run in every  render profiles functions to reset the page structure based on the selection
-function noRsultsPageHelper(){
+function noRsultsPageHelper() {
     $("#noResultsMessage").hide();
-    if(document.getElementById("selection").value === "mentors"){
+    if (document.getElementById("selection").value === "mentors") {
         $("#showMentors").show();
-    }else if(document.getElementById("selection").value === "mentees"){
+    } else if (document.getElementById("selection").value === "mentees") {
         $("#showMentees").show();
     }
 }
-function renderProfiles(mentorYear,menteeYear) {
+function renderProfiles(mentorYear, menteeYear) {
     noRsultsPageHelper();
     //show no results page if sorted mentor or mentee array is empty
-    if(document.getElementById("selection").value === "mentors" && mentorYear.length === 0){
+    if (document.getElementById("selection").value === "mentors" && mentorYear.length === 0) {
```

Comment:
```suggestion
    if (document.getElementById('mentorsCheckbox').checked == true && mentorYear.length === 0) {
```
like this?

Reply:
After you change every places like this. It will work, I checked it with my code & It worked.

Reply:
@YoshithaRathnayake

Reply:
ok

Reply:
Yes! It's working. Thank you very much @janithlahirukariyawasam